### ISSUE TITLE
jdbc mysql slower in active record 5?

### ISSUE THREAD

**Issue by terencechow** [zdennis/activerecord-import#740]
We recently upgraded a rails 4 app to rails 5 at my company. We noticed our import jobs were slower so we reverted the upgrade. Investigating it further, I believe the issue is that this library works slower for activerecord 5+ for jdbc mysql. Specifically, it is returning false for `supports_import?` and therefore falling back to the slower `connection.insert` method instead of the `connection.insert_many`.

This library adds methods (including `supports_import?`) to [ActiveRecord::ConnectionAdapters::MysqlAdapter](https://github.com/zdennis/activerecord-import/blob/master/lib/activerecord-import/active_record/adapters/jdbcmysql_adapter.rb#L4) for the jdbc mysql adapter.

However the activerecord-jdbcmysql-adapter for active record 5 points to [arjdbc/mysql](https://github.com/jruby/activerecord-jdbc-adapter/blob/master/activerecord-jdbcmysql-adapter/lib/active_record/connection_adapters/jdbcmysql_adapter.rb) which creates a class called [ActiveRecord::ConnectionAdapters::Mysql2Adapter](https://github.com/jruby/activerecord-jdbc-adapter/blob/master/lib/arjdbc/mysql/adapter.rb#L23). 

If I'm understanding this correctly, the method `supports_import?` is being added to the wrong class and jruby + mysql users are getting the inefficient `.insert` method instead of the more efficient `.insert_many`. 

I think the solution is really as simple as updating the class name in this library to Mysql2Adapter. I'm not 100% familiar with either the adapter or this library so if I am mistaken about this analysis please let me know. If you agree I can submit a pull request.

**Comment by terencechow** [comment]
Technically, this library works for jdbc mysql and active record 4 so changing the class name would represent a breaking change for anyone using this library and active record 4. As an alternative we could put two classes in jdbcmysql_adapter.rb so that it looks like the below. 

Let me know if you want to do that and I'll make a pull request

```
require "active_record/connection_adapters/mysql_adapter"
require "activerecord-import/adapters/mysql_adapter"

class ActiveRecord::ConnectionAdapters::MysqlAdapter
  include ActiveRecord::Import::MysqlAdapter
end

class ActiveRecord::ConnectionAdapters::Mysql2Adapter
  include ActiveRecord::Import::MysqlAdapter
end
```

**Comment by jkowens** [comment]
@terencechow I think you must be one of our few JRuby users 😄 

If you are interested in fixing this issue and creating a PR that would be very much appreciated!

**Comment by jkowens** [comment]
@terencechow can you test your app using our master branch? I updated the jdbc mysql adapter to just open the Mysql2Adapter. I'm hoping that works with both Rails 4 and 5.

**Comment by jkowens** [comment]
Released update in v1.2.0.
